benchmarks/bioid_ner_benchmark.py

```diff
@@ -0,0 +1,520 @@
+import os
+import json
+import pathlib
+from collections import defaultdict, Counter
+
+from functools import lru_cache
+import pandas as pd
+import xml.etree.ElementTree as ET
+
+from lxml import etree
+from tqdm import tqdm
+from datetime import datetime
+from typing import List, Tuple, Set, Dict, Optional, Iterable, Collection
+import click
+import pystow
+import gilda
+from gilda import ground
+from gilda.ner import annotate
+from gilda.grounder import logger
```

Comment:
Instead of this, we can do
```
import logging
logging.getLogger('gilda.grounder').setLevel('WARNING')
logger = logging.getLogger('bioid_ner_benchmark')
```

Reply:
On second thought the third line here may not be necessary if we use tqdm.write throughout